### ISSUE TITLE
Cargo test fail with test_cgroup

### ISSUE THREAD

**Issue by CaptainVincent** [containerd/runwasi#28]
**Not fail on every machines.**

I have success and fail machines both, but I am not sure what kind of information I could provide.
I could help test in my environment if you have any idea.
> So far I know if I  `sudo mkdir` under `/sys/fs/cgroup/memory`, they will generate different folder content between success and fail machine.

Run
```shell
cargo test --all test_cgroup --verbose
```

And I got those error log.
```
---- sandbox::cgroups::tests::test_cgroup stdout ----
Error: Others("failed to apply cgroup: could not open cgroup file /sys/fs/cgroup/relative/nested/containerd-wasm-shim-test_cgroup/memory.max: No such file or directory (os error 2)")

---- sandbox::cgroups::tests::test_cgroup stdout ----
running test with sudo: sandbox::cgroups::tests::test_cgroup
Error: Stdio(Kind(Other))
```

**Comment by cpuguy83** [comment]
This should be fixed by #33

**Comment by devigned** [comment]
Closing as it appears to be fixed in #33 (as @cpuguy83 mentioned). If this is still and issue, please reopen.

**Comment by devigned** [comment]
I was premature in closing this as the PR hasn't merged. Sry.
